### DISCUSSION TITLE
Exclude copy only backups

### DISCUSSION THREAD

**Discussion by paranoidDBA** [trimble-oss/dba-dash#948] [General]
Hi, is there a way to exclude copy-only backups from being checked? thanks

**Comment by DavidWiseman** [comment]
There isn't a way to exclude copy only backups at the moment.  I understand why you would want to exclude them - they are typically out of band backups that don't form part of your recovery strategy.  It might make sense for them to be excluded by default - though this would be a change in behaviour and some users might want to consider them.  Either way it would be useful to have a config option somewhere for this.
I don't tend to use copy only backups myself so they haven't been given much consideration.

**Reply by paranoidDBA** [reply]
ok thanks for the reply.
